src/CliqStateMachine_Parametric.jl

```diff
@@ -306,10 +330,18 @@ function waitForDown_ParametricStateMachine(csmc::CliqStateMachineContainer)
       end
       @error "Par-4, $(csmc.cliq.index): Exit with error state"
       return IncrementalInference.exitStateMachine
+    
+    elseif beliefMsg.status == :DOWNSOLVED 
```

Comment:
just check CSM uses lower case `:downsolved` -- I had a multihour debug issue on upper/lower case status the other day.  Qe can go back to enums whenever is best.

Reply:
Thanks, looks like I'm still on uppercase for up and down.